frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -1140,7 +1140,7 @@ export const createKnativeEventSourceSink = (
   }
   const eventSourceObj = _.omit(source, 'status');
   let sink = {};
-  if (NodeType.SinkUri === target.type?.nodeType) {
+  if (NodeType.SinkUri === target.kind) {
```

Comment:
`NodeType.SinkUri` has a value of `'sink-uri'`.

```suggestion
  if (URI_KIND === target.kind) {
```

Reply:
done